packages/web-components/src/components/video-player/video-player-container.ts

```diff
@@ -242,6 +242,14 @@ export const C4DVideoPlayerContainerMixin = <
         playerId,
         this._getPlayerOptions(backgroundMode)
       );
+      const { width, height } = await KalturaPlayerAPI.api(videoId);
+      videoPlayer.style.setProperty('--native-file-width', `${width}px`);
+      videoPlayer.style.setProperty('--native-file-height', `${height}px`);
+      videoPlayer.style.setProperty(
+        '--native-file-aspect-ratio',
+        `${width / height}`
```

Comment:
Nit: Might want to round this ourselves to the nearest thousandth or so, just to avoid repeating numbers (e.g. 1920x1080 resolves to 1.7777777777777777...). Browsers sometimes do weird things with long decimals in CSS.